Resources/views/menu.html.twig

```diff
@@ -0,0 +1,3 @@
+{% extends 'knp_menu.html.twig' %}
+
+{% block label %}{{ parent()|trans({}, item.getExtra('translation_domain', 'messages')) }}{% endblock %}
```

Comment:
so what would be the solution? copy the entire thing at https://github.com/KnpLabs/KnpMenu/blob/master/src/Knp/Menu/Resources/views/knp_menu.html.twig#L101 and try to translate safe_label or label and do |raw if we translated safe_label?